src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+import duke.exception.InvalidInputException;
+
+public class Deadline extends Task {
+    // This value indicates the version of this class when doing serialization
+    public static final long serialVersionUID = 01L;
+
+    protected String by;
```

Comment:
You might want to replace this with a more descriptive name since `by` can be understood as either the person who created the task or the deadline of the task

Reply:
Thanks for the advice. This has been fixed in the latest commit.